regression/contracts/function-calls-05/main.c

```diff
@@ -0,0 +1,27 @@
+#include <assert.h>
+#include <stdlib.h>
+
+static int add_operator(const int *a, const int *b)
+  __CPROVER_requires(a != NULL && b != NULL)
+    __CPROVER_ensures(__CPROVER_return_value > 0)
```

Comment:
Also, this wouldn't hold right? For `a == b == 0`?

Reply:
Yes, but it might be part of the specification, right? :)

Reply:
You're right! I removed the contracts for the this "add_operator". We should implement now the nondet assignments when enforcing contracts.